clang/test/Headers/riscv-sifive-header.c

```diff
@@ -0,0 +1,6 @@
+// RUN: %clang_cc1 -triple riscv64 -target-feature +v %s
+// REQUIRES: riscv-registered-target
+
+// expected-no-diagnostics
+
+#include <sifive_vector.h>
```

Comment:
Does this test case help verify we have `sifive_vector.h` installed?

Reply:
Lit test is run before install, how can we check it's properly installed?

Reply:
I think we can't :(